coverage-tests/configuration/python_overrides_universal.js

```diff
@@ -11,6 +11,18 @@
 
     // Failing on the universal server 1.10 and lower
     // check on new universal server version
-    "check window on mobile web android": {skip: true}
+    "check window on mobile web android": {skip: true},
 
+    // Temporarily disabled until generation fixed
+    'should abort after close with vg': {skipEmit: true},
+	'should abort unclosed tests': {skipEmit: true},
+    'should abort unclosed tests with vg': {skipEmit: true},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/eyes.sdk.python/pullRequest?prid=9121998)
```suggestion
    "should abort unclosed tests with vg": {skipEmit: true},
```